DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
I'd have to look into the SingleInstance option, since I haven't used that before, but from what I've seen, I believe the code I have currently is more flexible than using the SingleInstance method.

If it's configured as a single instance, it will replace all forwarders currently set on the server with the IPs in the IPAddress array. It wouldn't be able to just add/remove a single forwarder to/from an existing list of forwarders.